easypost.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rdoc', '~> 6.5'
   spec.add_development_dependency 'rspec', '~> 3.12'
   spec.add_development_dependency 'rubocop', '~> 1.49'
-  spec.add_development_dependency 'rubocop-rspec', '~> 2.19'
+  spec.add_development_dependency 'rubocop-rspec', '2.19'
```

Comment:
## Reviewer note:

the latest version of `rubocop-rspec` drop the support for Ruby 2.6 which we still use, pinning to `2.19` until we drop Ruby 2.7

Reply:
Please leave a comment in code describing why this is pinned, lest someone in the future tries to unpin it and has a bad day